src/types/table.ts

```diff
@@ -30,6 +34,15 @@ export interface TableHeader {
   useSortHandlerFunction?: boolean
 }
 
+export enum TableViewHeaderKeys {
+  ACTIONS = 'actions',
+}
+
+export interface TableViewHeader extends Omit<TableHeader, 'useSortHandlerFunction'> {
+  /** Must be unique for each column. If 'actions' value is provided, the column is treated as a column that only displays action dropdown menus for each row and no label (hideLabel: true) */
+  key: string | `${TableViewHeaderKeys}`
```

Comment:
issue: this type will not resolve property if more than `'actions'` is provided in the `TableViewHeaderKeys` - I think just type to a `string` here.